dependency-injection/2.0/_index.md

```diff
@@ -1,21 +1,21 @@
 ---
-title: "Jakarta Dependency Injection 2.0 (milestone)"
-date: 2020-05-18
+title: "Jakarta Dependency Injection 2.0"
+date: 2020-06-7
 summary: "Release for Jakarta EE 9"
 ---
 
 Jakarta Dependency Injection specifies a means for obtaining objects in such a way as to maximize reusability, testability and maintainability compared to traditional approaches such as constructors, factories, and service locators (e.g., JNDI).
 
 * [Jakarta Dependency Injection 2.0 Release Record]()
```

Comment:
Release record is missing. It should be https://projects.eclipse.org/projects/ee4j.cdi/releases/2.0

Reply:
@starksm64, Not sure why this item was resolved... Since DI is provided as part of the CDI project, then you should reference the CDI Release Record.  As it stands today, the rendering of the _index.md file looks like the Release Record is missing (which it is).  The easiest fix would be to just reference the CDI release record -- although there is no mention of the DI release in that release record.  An easy edit can correct that though.  Thanks.